app/controllers/matchings_controller.rb

```diff
@@ -20,32 +23,33 @@ def new
 
   # GET /projects/:project_id/matching/edit
   def edit
+    respond_to do |format|
+      if all_submitted_preferences?
+        #matching = Matching.where(project_id: @project.id).update_all(output_json: api)
+        if @matching.update(output_json: api)
+          format.html { redirect_to project_matching_path, notice: 'Successfully matched.' }
```

Comment:
This looks + works totally fine, but since we use flash['success'] everywhere else, we should keep it consistent here.